Content.Shared/Body/Systems/SharedBodySystem.Organs.cs

```diff
@@ -38,6 +40,10 @@ private void RemoveOrgan(Entity<OrganComponent> organEnt, EntityUid parentPartUi
             RaiseLocalEvent(organEnt, ref removedInBodyEv);
         }
 
+        if (TryComp(parentPartUid, out DamageableComponent? damageable)
+            && damageable.TotalDamage > 200)
```

Comment:
```suggestion
            && damageable.TotalDamage > 200)
```
Need to replace the 200 here with MobThresholdSystem.TryGetThresholdForState(uid, MobState.Dead, out var deadThreshold). Don't hardcode the assumption that all player characters die when they take 200 damage, we have a codebase where that is very likely not the case.

Reply:
![image](https://github.com/user-attachments/assets/f5bac323-782a-4d8a-8865-65776453a1ec)

Reply:
Oh thats handy. Though I was mostly intending for organs from badly damaged people to be useless until I get health/damageablecomponent simulations, not so much dead people.

Reply:
Oh wait this is not on a player character lol? Guh.